parlai/zoo/dodecadialogue/build.py

```diff
@@ -6,18 +6,28 @@
 """
 Pretrained models from The Dialogue Dodecathlon.
 
+The dodecadialogue_v2.tgz file contains a README.md and the dict for all models.
+
 https://arxiv.org/abs/1911.03768
 """
 from parlai.core.build_data import built, download_models, get_model_dir
 import os
 import os.path
 
 
-def download(datapath):
-    model_name = 'dodecadialogue'
-    mdir = os.path.join(get_model_dir(datapath), model_name)
-    version = 'v1.0'
-    if not built(mdir, version):
+def download(datapath, model_name):
+    mdir = os.path.join(get_model_dir(datapath), 'dodecadialogue')
+    dodeca_version = 'v2.0'
+    if not built(mdir, dodeca_version):
```

Comment:
Hmm, I know you allude to it at the top, but for clarity, I think it'd be worth leaving a comment here saying that we always download `dodecadialogue_v2` because that contains the README and a dict for all models, but that we only download the specified model file

Reply:
i'll add to the docstring